### ISSUE TITLE
fresh docker install crashes at 'ru' locale with 'ru' browser at admin search 

### ISSUE THREAD

**Issue by Sokharev** [helpyio/helpy#696]
What:
search in admin page not works if 'ru' locale enabled (and browser locale is 'ru')
WTR:
1. install fresh docker
2. enable 'ru' locale
3. have 'ru' locale browser
4. try search 'welcome' phrase in 'admin' search page
5. you'll see this in url:
http://my-new-install/admin/search/topic_search?utf8=%E2%9C%93&q=welcome&start_date=&end_date=&commit=%D0%9F%D0%BE%D0%B8%D1%81%D0%BA
and "Oops there has been an error... 
We log all errors and an engineers pager is going off somewhere!" page

**Comment by CGA1123** [comment]
@Sokharev Are you able to provide any of these logs?

**Comment by scott** [comment]
Yeah it would be helpful if you could find the error line in `development.log` or `production.log`
